aea/decision_maker/active.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""This module contains the pointers to the active decision maker class and auxilliary classes."""
+
+from aea.decision_maker.default import DecisionMaker  # noqa
+from aea.decision_maker.default import GoalPursuitReadiness  # noqa
+from aea.decision_maker.default import LedgerStateProxy  # noqa
+from aea.decision_maker.default import OwnershipState  # noqa
+from aea.decision_maker.default import Preferences  # noqa
```

Comment:
For now we just import the default dm. The idea is that the user can provide a path to a user defined dm which then gets loaded here and is therefore available for import from here.

Reply:
see above on using `aea/decision_maker/__init__.py` to achieve that

Reply:
Another idea: rename `aea.decision_maker.default.DecisionMaker` with `DefaultDecisionMaker` and then in in the `__init__.py`:

```python
from aea.decision_maker.default import DefaultDecisionMaker as DecisionMaker
```

Reply:
It should be clear however that if you only need the interface you have to import from `aea.decision_maker.base`, whereas if you need the concrete class you have to import from `aea.decision_maker.some_module_name.(Concrete)DecisionMaker`.

Not prepending some word like `Default` could cause some clashes with the `base` name. 
So either rename `.base.DecisionMaker` (`AbstractDecisionMaker`, `IDecisionMaker`, `ADecisionMaker` are some ideas) or rename the concrete classes